src/features/main_menu/_components/login/SignedIn.vue

```diff
@@ -99,6 +99,14 @@
                   label="On User Sign In"
                 />
               </v-col>
+              <v-col lg="4" cols="6">
+                <v-switch
+                  v-model="userProfile.SyncFrequency.onBulkDelete"
+                  hide-details
+                  color="accent"
+                  label="On Bulk Data Overwrite"
+                />
+              </v-col>
```

Comment:
Including an "On Bulk Data Overwrite" option for when users want to control whether a bulk delete or bulk import will automatically upload to the Cloud.